main.cpp

```diff
@@ -104,12 +107,25 @@ void BoulderProblem::Hold::printHoldInfo()
     std::cout << std::endl;
 }
 
+void BoulderProblem::Hold::printHoldInfo_()     // new function
+{
+    std::cout << "Hold Type: " << this->holdType;
+    std::cout << "\nHold Size: " << this->holdSize;
+    std::cout << "\nHold Height: " << this->holdHeight;
+    std::cout << std::endl;
+}
```

Comment:
just add `this->` to your previous function.   don't write a new one.  
you're only supposed to write new functions if you didn't have any `std::cout` statements in `main` that accessed member variables of your objects.
- [ ] remove this function
- [ ] revise previous function with same name.